spec/factories/case_attachments.rb

```diff
@@ -52,4 +52,12 @@
     preview_key { "#{SecureRandom.hex(16)}/request_previews/" +
                   "#{upload_group}/#{Faker::Internet.slug}.pdf" }
   end
+
+  factory :case_ico_decision, parent: :case_attachment do
+    type 'ico_decision'
```

Comment:
Didn't we just merge a PR to convert all such attributes from static to dynamic? (by putting them in `{...}`) (see #404)

Reply:
Good spot!!